core-bundle/src/Composer/ScriptHandler.php

```diff
@@ -70,8 +70,12 @@ private static function executeCommand(array $cmd, Event $event): void
             $command[] = $verbose;
         }
 
-        // Composer uses an old version of symfony/console which expects a string (see #1956)
-        $process = new Process(implode(' ', $command));
+        // Backwards compatibility with symfony/process <3.3 (see #1964)
```

Comment:
Is this 3.3 instead of 2.7 because people would use a `composer.phar` built for PHP7.2?

Reply:
It is `3.3` because any prior version does not handle an array.

Reply:
Ah my bad, misread the comment.